test/columns.test.py

```diff
@@ -482,6 +482,41 @@ def test_uda_format_unrecognized(self):
         """
 
 
+class TestUDAUUIDFormats(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Executed once before any test in the class"""
+        cls.t = Task()
+        cls.t.config("verbose", "nothing")
+        cls.t.config("uda.uda_uuid.label", "uda_uuid")
+        cls.t.config("uda.uda_uuid.type", "uuid")
+        cls.t.config("report.xxx.columns", "id,uda_uuid")
+
+        cls.t("add zero")
+        code, out, err = cls.t("_get 1.uuid")
+        cls.t("add uda_uuid:{} one".format(out))
+        code, out, err = cls.t("_get 2.uda_uuid")
+        cls.uda_uuid = out.strip()
+
+    def setUp(self):
+        """Executed before each test in the class"""
+
+    def test_uda_uuid_long(self):
+        """Verify formatting of 'uda_uuid.long' column"""
+        code, out, err = self.t("xxx rc.report.xxx.columns:id,uda_uuid.long")
+        self.assertIn(self.uda_uuid, out)
+
+    def test_uda_uuid_short(self):
+        """Verify formatting of 'uda_uuid.short' column"""
+        code, out, err = self.t("xxx rc.report.xxx.columns:id,uda_uuid.short")
+        self.assertIn(self.uda_uuid[:7], out)
```

Comment:
This would also match the long format. Maybe assert `self.uda_uuid[:7] + '\n'` is in output?